src/pages/index.js

```diff
@@ -2,37 +2,95 @@ import FormValidator from "../components/FormValidator.js";
 import Card from "../components/Card.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupConfirmDeletion from "../components/PopupConfirmDeletion.js";
 import Section from "../components/Section.js";
-import { initialCards } from "../utils/utils.js";
 import { formValidationConfig } from "../utils/utils.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
 import "./index.css";
 
```

Comment:
Please, delete `this._disableButton();` from line 77 in `FormValidator`

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.

Reply:
Ah, that makes more sense! I was wondering how to do this correctly because I noticed the button would immediately disable when clicked.